docs/Release_Notes.txt

```diff
@@ -0,0 +1,497 @@
+
+DOCS file changes
+-----------------
+ docs/README.md
+ Update build instructions  
+
+VERSIONS file changes
+----------------
+ versions/run.ver
+ Secure turn off SYNDAT for nam
+
+FIX code changes
+-------------------
+ No changes.
+
+Source code changes
+-------------------
+ No changes
+
+Output changes
+--------------
+ Generate new dumps:
+ *gmi1cr*, *snocvr*, *saldrn*, *subpfl*
+ Update dumps:
+ *gpsro*, *satwnd* and *nsstbufr (*nsstbufr is generatd only in global) 
+
+Compute Resource Information
+----------------------------
+ Memory usage was increased, from 60 to 120gb, when testing with triggers/jglobal_dump.wc2.pbs
+
+Disk Space Changes 
+------------------
+ The newly generated dumps ( *gmi1cr*, *snocvr*, *saldrn*, *subpfl* ) vary in size from 1Mb to 200Mb
+ depending on network, cycle time, time dump window. 0 size dumps are possible
+ The updated dumps (*gpsro*, *satwnd* and *nsstbufr) are up to ~25% larger in size
+ For examples, see: /lfs/h2/emc/stmp/iliana.genkova/CRON/RELv1.1.0/com
+ 
+Shared Software 
+---------------
+ obsproc needs modules bufr-dump and prepobs in order to run
+
+Libraries
+---------
+ See
+ versions/build.ver 
+ versions/run.ver 
+   
+Build Instructions
+------------------
+ See README.md
+
+Preimplementation Testing Requirements
+--------------------------------------
+ Run jobs/JOBSPROC_networkname_DUMP and jobs/JOBSPROC_networkname_DUMP_POST
+ for the following networkname values: GLOBAL, RAP, URMA, RTMA, CDAS
+
+Dissemination
+-------------
+ No change in dissemination. 
+
+Special Instructions
+--------------------
+ 1.Implement release/obsproc.v1.1.0 and release/bufr-dump.v1.1.0 at the same time
+ 2.Implement release/obsproc.v1.1.0 and release/bufr-dump.v1.1.0 before or at the same time as GFS v16.3 
+
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+obsproc v1.0.0        -> released to NCO on Dec 21,2021
+        v1.0.2        -> implemented Jun 28, 2022             
+
+Transition from WCOSS Cray and Phase3 to WCOSS2
+======================================
+https://docs.google.com/document/d/1U4pu4iSlLOnbNrcBqBlOU7bnNcTFfe_MpjV8uKN4S40/edit
+
+Where is the release tag on subversion/git/vlab? 
+ git clone https://github.com/NOAA-EMC/obsproc
+ git checkout release/obsproc.v1.0.0
+
+List of external software used (anything outside of your vertical structure), including compilers and version numbers for everything.
+Software used must be a minimal list of modules/versions specified per job:
+ hdf5_ver=1.10.6
+ netcdf_ver=4.7.4
+ bacio_ver=2.4.1
+ w3emc_ver=2.7.3
+ sp_ver=2.3.3
+ sigio_ver=2.3.2
+ nemsio_ver=2.5.2
+ bufr_ver=11.5.0
+ prod_util_ver=2.0.8
+ prod_envir_ver=2.0.5
+ cfp_ver=2.0.4
+ grib_util_ver=1.2.2
+ util_shared_ver=1.4.0
+ w3nco_ver=2.4.1
+ bacio_ver=2.4.1
+ bufr_dump_ver=1.0.0
+ gfs_ver=v16.2.0
+ omb_ver=v4.4.0
+ dictionaries_ver=v3.4.0
+
+List of all code/scripts modified with this release:
+A CMakeLists.txt
+A VERSION
+A build.sh
```

Comment:
This is not correct.  Also, again, why are all files changed needed to be listed in a Release Note?